src/keylessBackup/encryption.test.ts

```diff
@@ -0,0 +1,66 @@
+import crypto from 'crypto'
+import {
+  decryptPassphrase,
+  deriveKeyFromKeyShares,
+  encryptPassphrase,
+  getSecp256K1KeyPair,
+} from './encryption'
+
+describe("Encryption utilities using Node's crypto and futoin-hkdf", () => {
+  let keyshare1: Buffer
+  let keyshare2: Buffer
+  let passphrase: string
+
+  beforeEach(() => {
+    keyshare1 = crypto.randomBytes(16)
+    keyshare2 = crypto.randomBytes(16)
+    passphrase = 'testPassphrase'
```

Comment:
might be worth using a more realistic test passphrase, like an actual twelve word mnemonic